src/main/java/karuna/karuna_backend/ApplicationSetup/DefaultRolesInitializer.java

```diff
@@ -0,0 +1,49 @@
+package karuna.karuna_backend.ApplicationSetup;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import karuna.karuna_backend.Models.Role;
+import karuna.karuna_backend.Repositories.RoleRepository;
+
+/**
+ * Component responsible for initializing default roles in the database.
```

Comment:
Why need to think about comments in code, now its fine, becuase i look on it i know what code doing, but in clean code is writed about comments, and there is information about outdated comments, when u change code and forgot about change comment